App.jsx

```diff
@@ -1,91 +1,132 @@
-import React from 'react';
-import { NavigationContainer } from '@react-navigation/native';
-import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
-import { createNativeStackNavigator } from '@react-navigation/native-stack'
+import React, {useEffect} from 'react';
+import {NavigationContainer} from '@react-navigation/native';
+import {createMaterialBottomTabNavigator} from '@react-navigation/material-bottom-tabs';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
 import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
-import { Provider as PaperProvider } from 'react-native-paper';
-import { useColorScheme } from 'react-native';
-import { useMMKVStorage } from 'react-native-mmkv-storage';
+import {Provider as PaperProvider} from 'react-native-paper';
+import {useColorScheme} from 'react-native';
+import {useMMKVStorage} from 'react-native-mmkv-storage';
+import notifee, {
+  TimestampTrigger,
+  TriggerType,
+  RepeatFrequency,
+} from '@notifee/react-native';
 
 import MainView from './src/screens/MainView';
 import CalendarView from './src/screens/CalendarView';
 import AnalysisView from './src/screens/AnalysisView';
 import SettingsView from './src/screens/SettingsView';
 import AuthenticationView from './src/screens/AuthenticationView';
-import { CombinedDefaultTheme, CombinedDarkTheme } from "./src/theme";
-import { PIN_KEY, SettingsDB } from './src/storage/settings';
+import {CombinedDefaultTheme, CombinedDarkTheme} from './src/theme';
+import {PIN_KEY, SettingsDB} from './src/storage/settings';
 
 const App = () => {
   const scheme = useColorScheme();
 
   const Tab = createMaterialBottomTabNavigator();
   const Stack = createNativeStackNavigator();
 
-  const [pin, _] = useMMKVStorage(PIN_KEY, SettingsDB, "");
+  const [pin, _] = useMMKVStorage(PIN_KEY, SettingsDB, '');
+
+  async function onCreateTriggerNotification() {
+    const date = new Date(Date.now());
+    date.setHours(16);
+    date.setMinutes(11);
+
+    // Create a time-based trigger
+    const trigger: TimestampTrigger = {
```

Comment:
Ilmoitti tästä rivistä näin: "Type annotations can only be used in TypeScript files.ts(8010)"